src/style.css

```
@@ -0,0 +1,115 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
```

use `*` selector only to set `box-sizing`, since it should be the same for all elements. But not the specific styles as margins or padding, you should use tagnames for that
```suggestion
* {
  box-sizing: border-box;
}
```